pkg/controller/nodes/executor.go

```diff
@@ -214,17 +214,17 @@ func (c *nodeExecutor) execute(ctx context.Context, h handler.Node, nCtx *nodeEx
 	// check for timeout for non-terminal phases
 	if !phase.GetPhase().IsTerminal() {
 		activeDeadline := c.defaultActiveDeadline
-		if nCtx.Node().GetActiveDeadline() != nil {
+		if nCtx.Node().GetActiveDeadline() != nil && *nCtx.Node().GetActiveDeadline() > 0 {
 			activeDeadline = *nCtx.Node().GetActiveDeadline()
 		}
 		if c.isTimeoutExpired(nodeStatus.GetQueuedAt(), activeDeadline) {
 			logger.Errorf(ctx, "Node has timed out; timeout configured: %v", activeDeadline)
-			return handler.PhaseInfoTimedOut(nil, "active deadline elapsed"), nil
+			return handler.PhaseInfoTimedOut(nil, fmt.Sprintf("active deadline(%v) elapsed", activeDeadline)), nil
```

Comment:
the message is very weird, can we say, TimedOut?

Reply:
sure